dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -66,8 +67,11 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
                         log.error("Could not send pull request.");
                     }
                 });
+            } else {
+                imagesThatCouldNotBeProcessed.add(image);
```

Comment:
a) I like that we are now recording the images that didn't return results to this error summary list. 
b) We should move move the try catch from the util class to per my comment there. If you do that, I recommend having the try block start at line 52 and go until line 59.